lighthouse-core/test/config/default-config-test.js

```diff
@@ -30,7 +30,7 @@ describe('Default Config', () => {
 
     // And that they have the correct shape.
     opportunityResults.forEach(auditResult => {
-      assert.strictEqual(auditResult.details.type, 'opportunity');
```

Comment:
any reason for this change?

Reply:
yeah I failed this test while I was working and the error message you get is useless, this way it shows you what audit details you actually did get back so you know where to go fix